docs/painless/painless-contexts/painless-filter-context.asciidoc

```diff
@@ -23,4 +23,45 @@ query to include and exclude documents.
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To run this example, first follow the steps in
+<<painless-context-examples, context examples>>.
+
+This script allows to find all documents where the value of `doc['sold']`
+is `false`, and where the cost is less than 18.
+
+[source,Painless]
+----
+doc['sold'].value == false && doc['cost'].value < 18
+----
+
+To make a request more configurable, you can define `cost` as
+a script parameter. Submit the following script query request
+to filter all theatre seats for evening performances that are not
+sold yet, and where the cost is less than `params.cost`:
+
```

Comment:
I'd tweak this slightly to put the emphasis on parameterizing cost:

Defining `cost` as a script parameter enables the cost to be configured in the script query request. For example, 
the following request finds all available theatre seats for evening performances that are under $18.